client/app/components/Input/InputPassword.jsx

```diff
@@ -0,0 +1,51 @@
+// @flow
+import React from 'react';
```

Comment:
Let's also write a test for this new component! :) 
This is a [good example](https://github.com/ifmeorg/ifme/blob/master/client/app/components/Input/__tests__/InputSwitch.spec.jsx) of a similar test you want to write. Let me know if you need any help!